tests/unit/test_stages/base.py

```diff
@@ -1,7 +1,11 @@
 from unittest import TestCase
 
+from datetime import datetime
+
```

Comment:
@jbzdak nit: might make sense to remove this newline, for consistency; and the one above it too - the idea is to visually separate builtins and third party packages from stuff in this project. It might be not strictly enforced everywhere, but would be good to do so.